ste/mgr-JobMgr.go

```diff
@@ -961,7 +964,11 @@ func (jm *jobMgr) scheduleJobParts() {
 				go jm.poolSizer()
 				startedPoolSizer = true
 			}
-			jobPart.ScheduleTransfers(jm.Context())
+			getSourceCredential := jm.credential == nil
+			jobPart.ScheduleTransfers(jm.Context(), getSourceCredential, jm.credential)
```

Comment:
getSourceCredential is just a flag saying jm.credential is nil or not. Here we are passing down both of parameters here. Flag can be ignored and directly jm.credential can be checked internally?

Reply:
done